src/hub.html

```diff
@@ -1325,6 +1357,16 @@
         </a-entity>
         <a-entity id="scene-preview-node">
         </a-entity>
+        <!-- <a-entity 
+            chess-board="squareSize: 3; hideBoard: false; yCorrections: 0.1 -0.3 -0.6 -0.6 -1.05 -1.3; teleportPlayers: true;"
+            position="-10.3 -2.5 -2"
+            networked="template: #template-waypoint-avatar"
+        ></a-entity> -->
```

Comment:
Do we need to keep this?

Reply:
This will be fixed in the chess-refactor-fixes PR awaiting review.